src/components/MultiSelect/index.js

```diff
@@ -181,20 +201,23 @@ const MultiSelect = React.forwardRef((props, ref) => {
                     <RenderIf isTrue={!shouldRenderChips}>
                         <StyledPlaceholder>{placeholder}</StyledPlaceholder>
                     </RenderIf>
-                    <RenderIf isTrue={shouldRenderChips}>
-                        <Chips value={value} variant={chipVariant} onDelete={handleDelete} />
-                    </RenderIf>
+                    <RenderIf isTrue={shouldRenderChips}>{content}</RenderIf>
                 </StyledChipContainer>
-                <StyledButtonIcon
-                    title="Add"
-                    variant="neutral"
-                    size="small"
-                    icon={<PlusIcon />}
-                    onClick={handleTriggerClick}
-                    disabled={disabled}
-                    ref={triggerRef}
-                    tabIndex="-1"
-                />
+                <RenderIf isTrue={!!value && selectedCount > 0}>
```

Comment:
here we should use a const that explains the intention

Reply:
Done